notebooks/official/model_monitoring/get_started_with_model_monitoring_setup.ipynb

```diff
@@ -974,7 +941,7 @@
         "\n",
         "- `display_name`: The human readable name for the monitoring job.\n",
         "- `project`: The project ID.\n",
-        "- `region`: The region.\n",
+        "- `location`: The location.\n",
         "- `endpoint`: The fully qualified resource name of the `Vertex AI Endpoint` to enable monitoring.\n",
```

Comment:
Don't think you need the back tics. Also endpoint should be in lower case:

- Vertex AI endpoints

Reply:
Did required changes based on the feedback given on PR